package.json

```diff
@@ -107,14 +104,7 @@
     "bracketSpacing": true
   },
   "lint-staged": {
-    "*.json": [
-      "yarn prettier --write",
-      "git add"
-    ],
-    "*.js": [
-      "eslint --fix",
-      "yarn prettier --write",
-      "git add"
-    ]
+    "*.json": ["yarn prettier --write", "git add"],
+    "*.js": ["eslint --fix", "yarn prettier --write", "git add"]
```

Comment:
I wonder if me + someone else's prettiers are fighting on this JSON file. Likely fixed by when I prefixed the prettier in here with `yarn`